docs/domains/Domain.md

```diff
@@ -168,6 +175,12 @@ ServerPod describes the configuration for a Kubernetes pod for a server.
 | `annotations` | Map | The annotations to be attached to generated resources. |
 | `labels` | Map | The labels to be attached to generated resources. The label names must not start with 'weblogic.'. |
 
+### Istio
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `istioEnabled` | Boolean | True, if this domain is deployed under an Istio service mesh. Defaults to true. Not required. |
+
```

Comment:
Move this to the new Experimental section?

Reply:
It's already there.  This is the definition of the Istio object, which is only referenced in Experimental.

Reply:
David is going to be adding other fields to this object to address other comments, e.g. ports.